code/controllers/subsystem/economy.dm

```diff
@@ -21,11 +21,32 @@ SUBSYSTEM_DEF(economy)
 	///Multiplied as they go to all department accounts rather than just cargo.
 	var/bounty_modifier = 3
 
-	//MonkeStation Edit: /tg/ Mail Port
+	//Monkestation edit begin
```

Comment:
There is a pr on upstream porting tg mailing system, you might want to wait until it gets merged

Reply:
We actually already have it.
I ported it myself locally right before the upstream efforts started.

It wasn't a 1:1 port, so I didn't send it upstream